From bfa96fec44b7db78b43b9b8d20d7ca02616f70ed Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 13 Aug 2018 17:41:57 +0200
Subject: Fix: "warning: implicit declaration of function 'minor' / 'major'"
 errors

Fix the "warning: implicit declaration of function 'minor' / 'major'"
compiler warnings which ultimately lead to the following build error:

src/get_media_devices.c:522: undefined reference to `major'

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 src/get_media_devices.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/get_media_devices.c b/src/get_media_devices.c
index 619734ea..453b0677 100644
--- a/src/get_media_devices.c
+++ b/src/get_media_devices.c
@@ -23,6 +23,7 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <string.h>
 #include <stdlib.h>
 #include <malloc.h>
-- 
2.11.0

